### ISSUE TITLE
sensortag - support for the TI Sensortag 2

### ISSUE THREAD

**Issue by vielmetti** [node-red/node-red-nodes#107]
TI has a new version of their Sensortag. This issue is tracking support for that new version in the underlying library:

https://github.com/sandeepmistry/node-sensortag/issues/34

The current state of things (March 2015) is that there are patches available from TI that add some support for the Sensortag 2, and with those supplied patches installed the node-red-node-sensortag module does work (albeit without support yet for the new sensors) though support is not yet complete. The goal is to merge everything in together and that hasn't yet happened.

I don't know the best place to discuss this, but most of the details are on the above mentioned issue. There is a little bit of subtlety involved as the new device has a couple more sensors, and the driver code in node-sensortag tries to figure out what kind of device it's talking to. I think that can be all hidden from Node-RED.

**Comment by vielmetti** [comment]
There's a pull request here (please don't merge it! but it seemed to be the best way to share the change, it's not ready yet).

https://github.com/node-red/node-red-nodes/pull/108

**Comment by knolleary** [comment]
Hi, thanks for this.

AS per our contribution guidelines, we need a signed CLA before we can accept pull-requests - https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md#contributor-license-agreement

As a general comment, you can just post a link to your repo/branch/commit if you want us to look at something - the pull request isn't needed until the code is read to merge.

**Comment by vielmetti** [comment]
Thanks, I'll get the CLA done. I wasn't quite sure of the best way to proceed to notify you all, I'll hold off on further pull requests until everything else is ready.

**Comment by dceejay** [comment]
Closing this as have recently updated the sensorTag node to support the newer tag also...
(but still not multiple tags at once)

**Comment by vielmetti** [comment]
Thanks. I think that multiple tags (or multiple BLE devices in general) is a harder issue

**Comment by dceejay** [comment]
yes - I think there is some progress in the library but it's not a done deal... so as this issue was about tag2 and that now works we'll leave this closed and can open a new issue for multiple tags... if required.

**Comment by vielmetti** [comment]
There is an existing issue #109 that was closed, @dceejay - take a look at that and see if it's reopenable, or I can write up a new one that reflects the broader issue of managing multiple connections to multiple diverse BLE devices at the same time.

**Comment by vielmetti** [comment]
As noted on #109, IBM has this problem addressed in an IBM-IoT contribution at. https://github.com/ibm-iot/node-red-contrib-sensortag . (just to bring this issue to a satisfying close)
